apps/Makefile.in

```diff
@@ -92,10 +93,14 @@ install:
 	 cp $$i $(DESTDIR)$(OPENSSLDIR)/misc/$$i.new; \
 	 chmod 755 $(DESTDIR)$(OPENSSLDIR)/misc/$$i.new; \
 	 mv -f $(DESTDIR)$(OPENSSLDIR)/misc/$$i.new $(DESTDIR)$(OPENSSLDIR)/misc/$$i ); \
-	 done
-	@cp openssl.cnf $(DESTDIR)$(OPENSSLDIR)/openssl.cnf.new; \
-	chmod 644 $(DESTDIR)$(OPENSSLDIR)/openssl.cnf.new; \
-	mv -f  $(DESTDIR)$(OPENSSLDIR)/openssl.cnf.new $(DESTDIR)$(OPENSSLDIR)/openssl.cnf
+	 done;
+	@set -e; for i in $(CONFS); \
+	do \
+	(echo installing $$i; \
+	 cp $$i $(DESTDIR)$(OPENSSLDIR)/$$i.new; \
+	 chmod 644 $(DESTDIR)$(OPENSSLDIR)/$$i.new; \
+	 mv -f $(DESTDIR)$(OPENSSLDIR)/$$i.new $(DESTDIR)$(OPENSSLDIR)/$$i ); \
+	 done;
```

Comment:
Hmmm?  The semi in itself doesn't matter, you can still add lines below.  The semi is only needed if you want to add to the same shell expression, and in that case, you'll likely need a backslash as well.

I'd remove those redundant semis, they're likely to be confusing more than anything else.